operations/loyaltymemberships.md

```diff
@@ -217,6 +229,8 @@ Updates information about the specified loyalty memberships.
 | :-- | :-- | :-- | :-- |
 | `LoyaltyMembershipId` | string | required | Unique identifier of the loyalty membership. |
 | `LoyaltyProgramId` | [String update value](#string-update-value) | optional | Unique identifier of the loyalty program \(or `null` if the name should not be updated\). |
+| `IsPrimary` | [Bool update value](#bool-update-value) | optional | Boolean value defining the primary loyalty membership for the account. |
```

Comment:
Missing Code here.

Reply:
You can't update the membership code

![image](https://user-images.githubusercontent.com/22341887/199525432-215fd040-bb21-44e0-93ea-73fcb0642394.png)

Reply:
Code was added